mobile/components/ThreadItem/style.js

```diff
@@ -53,11 +54,24 @@ export const FacepileContainer = styled.View`
 export const EmptyParticipantHead = styled.Text`
   color: ${props => props.theme.text.alt};
   background: ${props => props.theme.bg.wash};
-  border-radius: 15px;
+  border-radius: ${props => (props.radius ? `${props.radius}px` : '15px')};
```

Comment:
We should handle all this styling at the mobile/components/Avatar level. `border-radius` for user avatars should always be 100%. for community avatars, it should always be 25%.

The only thing that should change between instances is size and whether or not it has status and link elements.

Reply:
I learned tonight that react native does some weird things with border radius; for circles you have to set the border radius to 50% of the width/height, otherwise it over-radius-es and becomes a diamond :P

Reply:
Oh also just note this is an empty participant head - not an avatar, so we'll need to treat it differently anyways :)